src/katsdpcontroller/fake_servers.py

```diff
@@ -284,13 +285,15 @@ def __init__(self, *args, **kwargs) -> None:
                     initial_status=Sensor.Status.NOMINAL,
                 )
             )
+            default_delays_str = str([0.0] * antennas)
             self.sensors.add(
                 Sensor(
                     str,
                     f"{beam_name}.delay",
-                    "The delay settings of the inputs for this beam: (loadmcnt "
-                    "<ADC sample count when model was loaded>, delay <in seconds>,"
-                    "phase <radians>, ...)",
+                    "The delay settings of the inputs for this beam. Each input has "
+                    "a delay [s] and phase [rad]: (loadmcnt, delay0, phase0, delay1,"
+                    "phase1, ...)",
+                    default=f"(0, {default_delays_str[1:-1]})",
```

Comment:
This string slicing is a bit icky. I think it's also wrong, because you need a delay AND a phase per antenna, so you need twice as many 0.0's as you have.

Suggestion (untested!):
```python
default_delays = (0,) + (0.0, 0.0) * antennas  # loadmcnt plus delay and phase per antenna
...
        default=str(default_delays),
```

Reply:
Argh, too right. Sure enough, your suggestion does the trick. Please and thank you!

Agreed Re: String slicing. I meant to leave a comment to say we're slicing to get rid of the brackets in the string expansion.